src/app/admin/services/courthouses/courthouses.service.ts

```diff
@@ -43,6 +63,22 @@ export class CourthouseService {
     return this.http.get<SecurityGroup[]>(`${GET_SECURITY_GROUPS_PATH}`);
   }
 
+  getCourthouseTranscriptionCompanies(): Observable<SecurityGroup[]> {
+    return this.http.get<SecurityRole[]>(`${GET_SECURITY_ROLES_PATH}`).pipe(
+      switchMap((securityGroups) => {
```

Comment:
securityGroups should be renamed to securityRoles or just roles here

Reply:
All changed